expyfun/_experiment_controller.py

```diff
@@ -1106,15 +1106,18 @@ def check_force_quit(self):
         """Check to see if any force quit keys were pressed."""
         self._response_handler.check_force_quit()
 
-    def text_input(self, stop_key='return', pos=[0, 0], color='white',
-                   font_name='Arial', font_size=24, wrap=True, units='norm',
+    def text_input(self, stop_key='return', instruction_string='Type '
+                   ' response below', pos=[0, 0], color='white',
+                   font_name='Arial', font_size=24,  wrap=True, units='norm',
```

Comment:
FYI, we normally add new keyword arguments to the *end* of the function signature (for backwards compatibility with user scripts that were passing args positionally).  In this case it's probably fine since `text_input` is so new, but now you know.

Reply:
Also, `instruction_string` appears to have two spaces between `Type` and `response`.

Reply:
Ooh good catch.